lib/parameters.py

```diff
@@ -8,7 +8,7 @@
 import itertools
 
 def format_value(value, **params):
-  from common import get_media
+  from .common import get_media
```

Comment:
This change is in the wrong commit. Please combine all "import" syntax changes into a single commit, together.